multinet/uploaders/multipart_upload.py

```diff
@@ -0,0 +1,44 @@
+"""Multinet uploader for multi-part uploaded files."""
+from base64 import b64encode
+
+from multinet import db, util
+
+from flask import Blueprint, request
+from webargs import fields
+from webargs.flaskparser import use_kwargs
+
+# Import types
+from typing import Any
+
+from multinet.errors import RequiredParamsMissing
+
+bp = Blueprint("uploads", __name__)
+bp.before_request(util.require_db)
+
+
```

Comment:
We need to have swagger docs for these endpoints, so that they can be tried out by developers by visiting the swagger page.

Reply:
And regarding this, I'd feel free to make a new section called "Uploads". I believe you'd add a new entry to this:

https://github.com/multinet-app/multinet-server/blob/2d07777b2acf3e9dcde4c6e29f620a78f6ac285d/multinet/swagger/template.yaml#L220-L230